pkg/collectors/devices/ptp_devices.go

```diff
@@ -18,8 +18,9 @@ type PTPDeviceInfo struct {
 }
 
 type DevDPLLInfo struct {
-	State  string `json:"state"`
-	Offset string `json:"offset"`
+	EECState  string `json:"EECState"`  //nolint:tagliatelle // Because EEC is dpll name
```

Comment:
It might be worth disabling tagliatelle on tags containing `EEC` and `PPC`, and anything else we are using. I don't think there's an "allowed words" option for it, sadly (https://golangci-lint.run/usage/linters/#tagliatelle).

Add an item to .golangci.yaml `exclude-rules:`

Reply:
Good point I'll look into this.